zyte_spider_templates/utils.py

```diff
@@ -0,0 +1,7 @@
+import re
+
+from scrapy.utils.url import parse_url
+
+
+def get_domain(url: str) -> str:
+    return re.sub(r"www.*?\.", "", parse_url(url).netloc)
```

Comment:
Would it make sense to have this function more conservative?  It looks like this one can change netlocs like `wwworld.example.com`, or `my.wwworld.com`, or `awww.com`. 

What are the real-world cases besides www.example.com which we should support?